webapp/src/components/NamesPage/ClaimNamePage/ClaimNamePage.tsx

```diff
@@ -0,0 +1,510 @@
+                    : nameInvalidType === NameInvalidType.HAS_SPACES
+                    ? t('names_page.has_spaces')
+                    : t('names_page.invalid_characters')}
+                </div>
+              ) : null}
+            </div>
+
+            <span
+              className={classNames(
+                styles.nameCost,
+                isInputFocus && styles.fadeOut
+              )}
+            >
+              {t('names_page.name_cost', {
+                mana: (
+                  <>
+                    <Mana inline /> 100 MANA
+                  </>
+                ),
+                network: (
+                  <span className={styles.nameCostNetwork}>
+                    {t('names_page.ethereum_mainnet_network')}
+                  </span>
+                )
+              })}
+              <Popup
+                content={t('names_page.dao_tooltip', {
+                  link: (
+                    <a
+                      href="https://decentraland.zone/governance/proposal/?id=a3bdc100-9b34-11ed-ae61-5f6dd0bf8358"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      {t('global.learn_more')}
+                    </a>
+                  )
+                })}
+                position="top center"
+                hoverable
+                mouseLeaveDelay={500}
+                trigger={
+                  <img
+                    src={infoIcon}
+                    alt="info"
+                    className={styles.informationTooltip}
+                  />
+                }
+                on="hover"
+              />
+            </span>
+          </div>
+          <div className={styles.ctasContainer}>
+            <h1>{t('names_page.why_names')}</h1>
+            <div className={styles.cardsContainer}>
+              {cards.map((card, index) => (
+                <div key={index} className={styles.card}>
+                  <div
+                    className={classNames(
+                      styles.whyImgContainer,
+                      card.className
+                    )}
+                  >
+                    <img src={card.image} alt={card.title} />
+                  </div>
+                  <div className={styles.whyTextContainer}>
+                    <span>{card.title}</span>
+                    <p>{card.description}</p>
+                  </div>
+                </div>
+              ))}
+            </div>
+            <div className={styles.cardsContainer}>
+              <div className={styles.nameTakenCard}>
+                <div className={styles.buttons}>
+                  <div>
+                    <img src={Chest} alt="Chest" />
+                  </div>
+                  <div>
+                    <h2> {t('names_page.ctas.name_taken.title')}</h2>
+                    <span> {t('names_page.ctas.name_taken.description')}</span>
+                    <Button onClick={() => onBrowse()}>
```

Comment:
Just a simplification that would prevent a re-render of the button:
```suggestion
                    <Button onClick={onBrowse}>
```

Reply:
can't use that, the onclick passes the event as the first parameter, the `onBrowse` doesn't expect any param
![image](https://github.com/decentraland/marketplace/assets/8763687/f4f9edc8-8228-4ef4-8915-714aeb69d089)